ldm/invoke/model_cache.py

```diff
@@ -337,14 +334,15 @@ def preamble(self):
         '''
         Returns the preamble for the config file.
         '''
-        return '''# This file describes the alternative machine learning models
-# available to InvokeAI script.
-#
-# To add a new model, follow the examples below. Each
-# model requires a model config file, a weights file,
-# and the width and height of the images it
-# was trained on.
-'''
+        return textwrap.dedent('''
```

Comment:
The problem with textwrap.dedent is, in this way, the way it is implemented adds new lines to top and bottom
which can be cleaned up using strip, which wastes quite a bit of resources

it's probably better to avoid using textwrap.dedent in the first place

Reply:
using a backslash \ solves the problem: https://docs.python.org/3/library/textwrap.html#textwrap.dedent